src/components/pacbio/PacbioPoolLibraryEdit.vue

```diff
@@ -145,14 +145,21 @@ export default {
     ...mapMutations(['updateLibrary']),
     ...mapActions(['applyTags']),
     hasErrors(attribute) {
-      if (!this.library.errors) {
-        return null
+      //Check any errors exist in library for the given attribute
+      if (!this.library.errors || !this.library.errors[attribute]) {
+        return false
       }
-      return !this.library.errors[attribute]
+      return true
     },
     errorsFor(attribute) {
       return this.library.errors && this.library.errors[attribute]
     },
+    attributeValueExists(attribute) {
```

Comment:
again here you can do optional chaining.
`return this.library?.[attribute]?.length > 0`